packaging/src/docker/README.md

```diff
@@ -210,3 +210,54 @@ docker compose down
     select count(distinct a) from hive_example;
     select sum(b) from hive_example;
   ```
+
+#### Hive Schema Tool
```

Comment:
nit: How about change the title to `Information Schema`

Reply:
- I am leaning towards `Sys Schema`. HiveServer2 provides multiple databases including `information_schema`.